api/public/v2/repo/views.py

```diff
@@ -44,7 +44,15 @@ class RepositoryViewSet(
     def get_queryset(self):
         return super().get_queryset().with_recent_coverage()
 
-    @extend_schema(summary="Repository list")
+    @extend_schema(
+        summary="Repository list",
+        parameters=[
+            OpenApiParameter(
```

Comment:
Inspiration from this thread: https://github.com/tfranzel/drf-spectacular/issues/267